src/SwedbankPay.Sdk.Tests/TestHelpers/TestHelper.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Microsoft.Extensions.Configuration;
+
+namespace SwedbankPay.Sdk.Tests.TestHelpers
+{
+    public class TestHelper
+    {
+
+        public static IConfigurationRoot GetIConfigurationRoot(string outputPath)
+        {
+#if DEBUG
+            return new ConfigurationBuilder()
+                .SetBasePath(outputPath)
+                .AddJsonFile("appsettings.json", true)
```

Comment:
Consider using either appsettings or environment variables, but not both.

Reply:
This is for solving how devops fetches the e.g. token so we don't need to have it in the appsettings.json file in git.